lib/controllers/books.js

```diff
@@ -0,0 +1,14 @@
+const { Router } = require('express');
+const Book = require('../models/Book');
+
+module.exports = Router()
+  .get('/:id', async (req, res) => {
+    const book = await Book.getById(req.params.id);
+    res.json(book);
+})
```

Comment:
It looks like the CI failure eslint is throwing, points us to this line - try adding two spaces before the closing curly and paren.